pcap-linux.c

```diff
@@ -6945,6 +6950,26 @@ pcap_set_protocol(pcap_t *p, int protocol)
 	return (0);
 }
 
+int
+
+pcap_set_fanout(pcap_t *handle, int flags, int group_id)
```

Comment:
`flags` is actually a type and flags, as least as I read `packet_setsockopt()` and `fanout_add()` in `af_packet.c`.  Either it should be called `type_flags`, or there should be three arguments - `type`, `flags`, and `group_id` - which are combined in the argument to `setsockopt()`.